app/mixins/search_resource.js

```diff
@@ -44,12 +44,20 @@ export default Ember.Mixin.create({
   },
 
   onSearchTextChange: Ember.observer("searchText", function() {
-    if (this.get("searchText").length > this.get("minSearchTextLength")) {
-      this.rerenderOrShowNewResult();
+    const searchTextLength = this.get("searchText").length;
```

Comment:
What is the use of this const variable?

Reply:
Yeah I think it's not required here @abulsayyad123 .

Reply:
removed !!